tax/totals.go

```diff
@@ -232,6 +232,11 @@ func (rt *RateTotal) percentagesMatch(c *Combo) bool {
 	} else if rt.Surcharge != nil {
 		return false
 	}
+
+	if rt.Percent == nil {
+		return c.Percent == nil
```

Comment:
Wouldn't this just be:
```suggestion
		return true
```